<html>
<head>
<title>HERV regulatory elements database</title>
<link rel=stylesheet type="text/css" href="static/style.css">
</head>
<body>
<script src="static/plotly.min.js"></script>
<div id="wrap">
  <div id="title"><h1>HERV Regulatory Element Database</h1></div>
  <div id="contents">
    <div id="left">
      <div id="param_box">
        Parameters for TFBS and Motif</br>
        Database:
        <select id="db1">
          <option value="Roadmap|ENCODE">Roadmap and ENCODE</option>
          <option value="Roadmap">Roadmap</option>
          <option value="ENCODE">ENCODE</option>
        </select></br>
        <input id="hcre1" type="checkbox" name="hcre" value="hcre" checked>Only HCRE</input></br>
        
        <form id="z_score_mode">
          <input type="radio" name="z_score_mode" value="Count" checked>Count based Z score</input></br>
          <input type="radio" name="z_score_mode" value="Depth">Depth based Z score</input>
        </form>
        Z score &gt;= <input id="z_score1" value="3"></input></br>
        Show limit = <input id="limit1" value="10"></input></br>
        </br>
        Parameters for DHS</br>
        <input id="represent" type="checkbox" name="represent" value="represent" checked>Only representative cell types</input></br>
        Z score &gt;= <input id="z_score2" value="3"></br>
        Show limit = <input id="limit2" value="10"></br>
        <br>
        Parameters for download</br>
        <input id="merge_cell_types" type="checkbox" checked>Merge cell types</input></br>
      </div>
      <div id="herv_menu"></div>
    </div>
    <div id="right">
      <div id="introduction">
        <h2>Introduction</h2>
        <p>Human endogenous retroviruses and other LTR type retrotransposons (HERV/LRs) have their own regulatory elements possibly affecting on transcriptions of host genes. dbHERV-REs is a database for regulatory elements of these HERV/LRs. The dbHERV-REs particularly focuses on HERV/LRs Conserved Regulatory Elements (HCREs), regulatory elements conserved among HERV/LRs copies belonging to the same type of HERV/LRs. Besides above, this database provides phylogenetic tree of each HERV/LR type, and so you can compare phylogenetic subgroup in a particular HERV/LR type and presence or absence of HCREs. If you want to know about this database in detail, please check our paper (REF).</p>
      </div>
      <div id="text_body">
        <h1>HERV Regulatory Elements of <span class="herv_name"></span></h1>
        <p><a id="dl_hcre" class="download_anchor">Download</a> the positions of transctiption factor binding sites for the consensus sequence of <span class="herv_name"></span>.</p>

        <h2>Metaplot of HERV-TFBSs</h2>
        <p>A metaplot of HERV-TFBSs on a consensus sequence of the HERV/LR type. The x-axis indicates nucleotide positions of the consensus sequence. The y-axis indicates number of HERV/LR copies harboring HERV-TFBSs at that position.</p>
        <div id="tfbs_depth_graph"></div>
        <p><a id="dl_herv_tfbs_position">Download</a> the human genome loci of <select id="tf_select_1" anchor_id="dl_herv_tfbs_position"><option value="all">all transcription factors</option></select> binding sites those overlap with <span class="herv_name"></span>.</p>

        <h2>Metaplot of TF-binding motifs in TFBSs</h2>
        <p>A metaplot of TF-binding motifs present in HERV-TFBSs on consensus sequence of the HERV/LR type. The x-axis indicates nucleotide positions of the consensus sequence. The y-axis indicates number of HERV/LR copies harboring the binding motifs at that position. Peaks of TF-binding motifs corresponding to HCREs are shown with asterisks and motif names.</p>
        <div id="motif_depth_graph"></div>
        <p><a id="dl_hcre_position">Download</a> the human genome loci of <select id="tf_select_2" anchor_id="dl_hcre_position"><option value="all">all transcription factors</option></select > those conserved on <span class="herv_name"></span>.</p>

        <h2>Metaplot of HERV-DHSs</h2>
        <p>A metaplot of HERV-DHSs (overlaps between HERV/LRs and DHSs) on consensus sequence of the HERV/LR type. The x-axis indicates nucleotide positions of the consensus sequence. The y-axis indicates number of HERV/LR copies harboring HERV-DHSs at that position.</p>
        <div id="dhs_depth_graph"></div>
        <p><a id="dl_dhs_position">Download</a> the human genome loci of DNase Hypersensitive Sites those overlap with <span class="herv_name"></span>.</p>

        <h2>Overlaps of HERV/LRs with each chromatin state</h2>
        <p>Proportion of HERV/LR copies overlapped to each chromatin state predicted by chromatin segmentation. TSS; promoter region including TSS, PF; predicted promoter flanking region, E; enhancer, WE; weak enhancer or open chromatin cis regulatory element, CTCF; CTCF enriched element, T; transcribed region, R; repressed or low activity region.</p>
        <div id="chromatin_state_graph"></div>
        <div id="heat_maps"></div>
      </div>
    </div>
  </div>
</div>
<script src="static/main.js"></script>
</body>
</html>
